docs/v3/source/includes/resources/processes/_list_for_app.md.erb

```diff
@@ -40,12 +40,13 @@ Name | Type | Description
 **updated_ats** (*experimental*)| _[timestamp](#timestamps)_ | Timestamp to filter by. When filtering on equality, several comma-delimited timestamps may be passed. Also supports filtering with [relational operators](#relational-operators-experimental)
 
 #### Permitted roles
- |
+Roles | Notes |
 --- | ---
 Admin |
 Admin Read-Only |
 Global Auditor |
 Org Manager |
 Space Auditor |
 Space Developer |
+Space Application Developer | Experimental
```

Comment:
I haven't looked through this PR extensively yet, but just at first glance I believe this should actually be `Space Application Supporter` at this stage, and we've been aiming to alphabetize the roles. Let me know if I missed something there